src/test/java/fi/helsinki/cs/tmc/core/updates/ExerciseUpdaterTest.java

```diff
@@ -20,19 +20,12 @@
 import fi.helsinki.cs.tmc.core.testhelpers.ExampleJson;
 import fi.helsinki.cs.tmc.core.testhelpers.builders.ExerciseBuilder;
 
-import com.google.gson.Gson;
-
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
-import org.mockito.Mockito;
-
-import java.io.File;
-import java.io.FileWriter;
 import java.io.IOException;
+import java.net.URI;
 import java.net.URISyntaxException;
-import java.nio.file.Paths;
```

Comment:
Go to Tools | Options | Editor | Formatting, select Language: "Java", Category: "Imports" and there's an "Import Layout" section at the bottom. You'll probably still need to manually add newlines between the "third party" import groups, tho.

The checkstyle should warn about incorrect import order but there's currently an issue with our checkstyle config: testmycode/tmc-core#80.